RELEASE_NOTES.rst

```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
```

Comment:
Please look at your patch *very carefully*.
You have moved "` for C,`" to the previous line.
Please undo that change.